Makefile

```diff
@@ -70,6 +70,11 @@ nix-shell:
 # must be included after the default target
 -include $(BUILD_SYSTEM_DIR)/makefiles/targets.mk
 
+ifeq ($(filter $(mkspecs),android ios),$(mkspecs))
+-include mobile/scripts/EnvVariables.mk
+-include mobile/scripts/Common.mk
+endif
```

Comment:
This won't be necessary if you call `status-go` Makefile from `mobile/Makefile`.